### PR TITLE
Add --skip-build flag for init command #144

### PR DESCRIPTION
**Pull request by kohensu**
pull request for #144

### CODE REVIEW

File: src/command.rs [hunk 1]
```diff
@@ -145,111 +151,202 @@ pub fn run_wasm_pack(command: Command, log: &Logger) -> result::Result<(), Error
...
+    ) -> Init {
+        Init {
+            crate_path: set_crate_path(path),
+            scope,
+            skip_build,
+            disable_dts,
+            target,
+            crate_name: None,
+        }
+    }
```
Comment (ashleygwilliams) [inline src/command.rs]: would like a line here :)

File: src/command.rs [hunk 2]
```diff
@@ -145,111 +151,202 @@ pub fn run_wasm_pack(command: Command, log: &Logger) -> result::Result<(), Error
...
+    pub fn process(&mut self, log: &Logger) -> result::Result<(), Error> {
+        let steps: Vec<fn(&mut Init, &Step, &Logger) -> result::Result<(), Error>> =
+            if self.skip_build {
```
Comment (ashleygwilliams) [inline src/command.rs]: (ugh GH didn't post this comment in my original review)

so- i think we might be exploring even more "build" modes (see #153 ) - so (and i am likely overengineering here) but perhaps an enum and a match statement would work best here. would love to hear your thoughts!

File: src/command.rs (hunk 1 above)
Comment (kohensu) [reply src/command.rs]: oups

File: src/command.rs (hunk 2 above)
Comment (kohensu) [reply src/command.rs]: I did the modification this modification in the PR, if it is not like you what you had in mind I will revert it.

**Comment by ashleygwilliams** [comment]
also- just as a heads up we are probably landing 2 PRs today which means this will likely need a rebase, but i'd like to prioritize getting this in after that so that we don't have too much rebasing that needs to happen :)

**Comment by kohensu** [comment]
ok, I will do the rebase, the test and the doc

**Pull request merged**